src/pages/Root.tsx

```diff
@@ -1,57 +1,146 @@
 import { Outlet, Link } from "react-router-dom";
 import { ROUTES } from "../routes";
+import { Header } from "../components/Header";
+import { Logo } from "../components/Logo";
+import { Button } from "../components/Button";
+import "./navbar.css";
+import { Title } from "../components/Title";
+import mapa from "../assets/mapa.jpg";
+import { BlogCards } from "../components/BlogCards";
+import strava from "../assets/granule.jpg";
+import pohyb from "../assets/mapadog.jpg";
+import denik from "../assets/diar.jpg";
 
-const Root = () => {
+export const Root = () => {
 	return (
 		<>
 			<div id="sidebar">
-				<h1>React Router Contacts</h1>
-				<div>
-					<form id="search-form" role="search">
-						<input
-							id="q"
-							aria-label="Search contacts"
-							placeholder="Search"
-							type="search"
-							name="q"
-						/>
+				<nav className="navbar py-2 rounded-xl mt-5 mx-2 bg-darkGreen">
+					<div className="flex justify-between pl-4 pr-4 ">
+						<Logo />
 
-						<div id="search-spinner" aria-hidden hidden={true} />
-						<div className="sr-only" aria-live="polite"></div>
-					</form>
-					<form method="post">
-						<button type="submit">New</button>
-					</form>
-				</div>
-				<nav>
-					<ul>
-						<li>
-							<Link to={ROUTES.index()}>Domů</Link>
-						</li>
-						<li>
-							<Link to={ROUTES.about()}>O mně</Link>
-						</li>
-						<li>
-							<Link to={ROUTES.maps()}>Mapy</Link>
-						</li>
+						<ul className="flex bg-darkGreen">
```

Comment:
Tohle bych se nebál udělat jako NAvbar komponentu a NavItem třeba abycs tam nemusela psát ty stejný kod pořád dokola... Viděl bych to klidně tak, že bys měla komponentu Navbar, která by přijímala pole objektů, kde ten jeden objekt by byl tvůj link. [{link: Routes.index(), text: "Home" }, ...]